tool/net/redbean.c

```diff
@@ -1915,6 +1915,26 @@ static int LuaGetZipPaths(lua_State *L) {
   return 1;
 }
 
+static int LuaSetHandler(lua_State *L) {
+  int t = lua_type(L, 1);
+
+  if(t == LUA_TFUNCTION) {
+    lua_setglobal(L, "Handler");
+    lua_pushboolean(L, true);
+  } else {
+    lua_pushboolean(L, false);
```

Comment:
Lua doesn't tend to raise errors, except in very rare cases. For example, you'd expect `tonumber` to return `nil`, rather than raising an error. If you return a boolean, then people who want to be lazy can wrap it in an assert, but those with more complicated cases can build a series of fallbacks.